### ISSUE TITLE
Show developer tools in the tutorials

### ISSUE THREAD

**Issue by emschwartz** [interledger-deprecated/interledger#4]
I think we should show people how to use the `ilp` module in the tutorials, since that will probably be the entry point for most developers. Most people probably won't run connectors so I'd emphasize that a bit less than the client experience.

**Comment by michielbdejong** [comment]
Yes! In the [updated tutorials](https://github.com/interledger/tutorials/pulls) I used LPI and plugins, so that people can at the same time learn Interledger, and InterledgerJS, so to speak.

I didn't use the ILP module yet, that part of the InterledgerJS spec can be introduced in parallel in the same way, when we introduce the higher-level concepts of quoting, PSK, SPSP, etc.
